nuklai/chain.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "nuklai",
+  "chain_type": "cosmos",
+  "status": "live",
+  "network_type": "testnet",
```

Comment:
Please relocate this `nuklai` directory into the `/testnets/` directory, and rename the directory and all occurrences of `chain_name` to: the project name + "testnet", e.g., nuklaitestnet (which implies that the mainnet will be just nuklai)